aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -224,6 +254,38 @@ resource "aws_elasticsearch_domain" "example" {
 `, randInt)
 }
 
+func testAccESDomainConfig_ClusterUpdate(randInt int) string {
+	return fmt.Sprintf(`
+resource "aws_elasticsearch_domain" "example" {
+  domain_name = "tf-test"
+
+  advanced_options {
+    "indices.fielddata.cache.size" = 80
+  }
+
+  ebs_options {
+    ebs_enabled = true
+		volume_size = 10
+		
+  }
+
+  cluster_config {
+    instance_count = %d
```

Comment:
This should be certainly static number, we don't want to be spinning up an _unbound_ random number of instances 💸 😃 

Randomization is however certainly desired, but only in places that are unique, like `domain_name` see other tests in this file.